### DISCUSSION TITLE
What Dakota GUIs are available?

### DISCUSSION THREAD

**Discussion by thenectarine** [snl-dakota/dakota#147] [Q&A]
Dakota comes with the Dakota GUI, see figure 1. Is this the only GUI available to edit Dakota input files?

I have been mostly relying on a text editor to edit Dakota input files, so I'm curious to see what else is out there.

Figure 1 - Dakota GUI
![dakota_gui](https://github.com/user-attachments/assets/44cdb233-f07f-4f71-9710-e205d26bb3d6)

**Comment by jadamstephens** [comment]
A number of organizations have developed graphical frontends for Dakota for their internal use, but I'm not aware of any publicly-available, open-source GUIs for Dakota other than the one we maintain.

If anyone knows of one, please chime in.
